runtime/cspell.json

```diff
@@ -0,0 +1,177 @@
+{
+  "version": "0.2",
+  "language": "en",
+  "words": [],
+  "ignoreWords": [
+    "anyfunc",
+    "argty",
+    "assemblyscript",
+    "bulkmemory",
+    "chainlist",
+    "cranelift",
+    "demangle",
+    "dereferencable",
+    "dylib",
+    "emtests",
+    "Emscripten",
+    "externref",
+    "funcenv",
+    "funcref",
+    "funcrefs",
+    "funcs",
+    "Ggas",
+    "Greenwasm",
+    "hypercall",
+    "importobject",
+    "Kgas",
+    "libcall",
+    "libcalls",
+    "lightbeam",
+    "Mgas",
+    "Mprotect",
+    "mprotect",
+    "mremap",
+    "multivalue",
+    "munmap",
+    "nearvm",
+    "nondet",
+    "PGAS",
+    "petagas",
+    "probestack",
+    "pwasm",
+    "reftype",
+    "reftypes",
+    "reloc",
+    "relocs",
+    "seccomp",
+    "singlepass",
+    "spectest",
+    "spectests",
+    "srcloc",
+    "symbolicate",
+    "symbolication",
+    "teragas",
+    "terragas",
+    "TGAS",
+    "threadsafe",
+    "trapcode",
+    "traphandlers",
+    "uninit",
+    "vmcontext",
+    "vmctx",
+    "VMLogic",
+    "vmmem",
+    "vmmemory",
+    "vmoffsets",
+    "vmstate",
+    "wasi",
+    "wasmer",
+    "wasmprinter",
+    "wasms",
+    "wasmtime",
```

Comment:
Crate names such as `wasmprinter`, `wasmtime` (consult Cargo.toml dependencies) should be be considered valid words project wide (moved to the top-level cspell.json.) Don't miss any instances.